packages/blob/src/index.ts

```diff
@@ -94,7 +93,11 @@ export function isBlobURL( url ) {
  * @param {BlobPart} content     File content (BufferSource | Blob | string).
```

Comment:
Eslint complained here about the param type, but this has been converted to TypeScript so we need to remove all the types from JSDoc.

Reply:
@sirreal please review now I have removed the param type.